src/pages/understand-stacks/running-testnet-node.md

```diff
@@ -23,6 +23,29 @@ This tutorial will walk you through the following steps:
 
 ## Requirements
 
+In order to run a node, some software and hardware requirements need to be considered.
+
+### Hardware
+
+Running a node has no specialized hardware requirements. People were successful at running a node on Raspberry Pis, for instance.
+Minimum requirements are moving targets due to the nature of the project and some factors should be considered:
+
+- compiling node sources locally requires processing and storage resources
+- replicating the chainstate requires significant memory allocation (1,3+ GB)
+- chainstate size grows over time, increasing the memory requirements
+
+With these considerations in mind, we suggest hardware based on a general-purpose specification provided by GCP, the [E2 machine standard 2](https://cloud.google.com/compute/docs/machine-types#general_purpose):
```

Comment:
Perhaps add an equivalent AWS instance as well for completeness? It's not like we're recommending people use GCP or any specific vendor.

Reply:
great point, adding that as well